aws/README.md

```diff
@@ -1,89 +1,177 @@
 # AWS Command Set
 
-Interact with Amazon Web Services directly from Slack using Nimbella Commander.
+Interact with Amazon Web Services from messaging platforms like Slack, Mattermost & Microsoft Teams using [Nimbella Commander](https://nimbella.com/product/commander).
+
+- [Installation](#Installation)
+- [Requirements](#Requirements)
+- [Commands](#Commands)
+- [Usage](#Usage)
+- [Support](#support)
 
 ## Installation
 
-Please make sure you've Commander in your [Slack team](https://slack.com/apps/AS833QXL0-nimbella-commander) before executing this Slash command.
+Please make sure you've Commander installed in your [Slack workspace](https://slack.com/apps/AS833QXL0-nimbella-commander) before executing this Slash command.
```

Comment:
Please make sure Commander is installed in your [Slack workspace](https://slack.com/apps/AS833QXL0-nimbella-commander) before executing this Slash command.